src/polymer/docs.ts

```diff
@@ -350,6 +352,10 @@ export function featureElement(features: FeatureDescriptor[]):
         'The properties reflected here are the combined view of all ' +
         'features found in this library. There may be more properties ' +
         'added via other libraries, as well.',
+    observers: [],
```

Comment:
This shouldn't be necessary, they're set in the ElementDescription. If that's not working correctly (due to Object.assign?) can you fix it there?